jazz_apigee-proxy-aws/README.md

```diff
@@ -0,0 +1,41 @@
+Common Lambda function that is invoked from external API proxies(like Apigee) to invoke the lambda containing the logic. This lambda acts as the interface for other user created lambdas.
+
+Sample input
+
+{
+  "region": "us-east-1",
+  "functionName": "functionName",
+  "body": "{\"test\":\"body\"}",
+  "httpMethod": "POST",
+  "queryStringParameters": {
+    "foo": "bar"
+  },
+  "headers": {
+    "Via": "1.1 08f323deadbeefa7af34d5feb414ce27.cloudfront.net (CloudFront)",
+    "Accept-Language": "en-US,en;q=0.8",
+    "CloudFront-Is-Desktop-Viewer": "true",
+    "CloudFront-Is-SmartTV-Viewer": "false",
+    "CloudFront-Is-Mobile-Viewer": "false",
+    "X-Forwarded-For": "127.0.0.1, 127.0.0.2",
+    "CloudFront-Viewer-Country": "US",
+    "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
+    "Upgrade-Insecure-Requests": "1",
+    "X-Forwarded-Port": "443",
+    "Host": "1234567890.execute-api.us-east-1.amazonaws.com",
+    "X-Forwarded-Proto": "https",
+    "X-Amz-Cf-Id": "cDehVQoZnx43VYQb9j2-nvCh-9z396Uhbp027Y2JvkCPNLmGJHqlaA==",
+    "CloudFront-Is-Tablet-Viewer": "false",
+    "Cache-Control": "max-age=0",
+    "User-Agent": "Custom User Agent String",
+    "CloudFront-Forwarded-Proto": "https",
+    "Accept-Encoding": "gzip, deflate, sdch"
+  },
+  "pathParameters": {
+    "proxy": "path/to/resource"
+  },
+  "method": "POST",
+  "stageVariables": {
```

Comment:
Not required. 'Stage' is specific to AWS API Gateway service, we can remove this from here.